node-packages/commons/src/tasks.ts

```diff
@@ -1148,7 +1148,15 @@ export const createMiscTask = async function(taskData: any) {
           // Handle setting up the configuration for a restic restoration task
           const restoreName = `restore-${R.slice(0, 7, taskData.data.backup.backupId)}`;
           // generate the restore CRD
-          const restoreConf = restoreConfig(restoreName, taskData.data.backup.backupId, makeSafe(taskData.data.project.name))
+
+          let baasBucketName = ""
+          for (let variable of osResult.envVariables) {
+            if (variable.name == "LAGOON_BAAS_BUCKET_NAME") {
+              baasBucketName = variable.value
+            }
+          }
+          // Parse out the baasBucketName for any migrated projects
```

Comment:
Is this a comment related to this PR? migrations weren't mentioned in the description. If so, I'd put this above this block of code. Same comment for all the other instances of this comment.